package.json

```diff
@@ -10,7 +10,8 @@
     "preview": "vite preview",
     "format": "prettier --write .",
     "test": "vitest",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "check-tsc": "tsc"
```

Comment:
Change the name from check-tsc to tsc-check and add --noEmit to the script.

```bash
"tsc-check": "tsc --noEmit"
```

With this flag the transpilation js files are not generated and we save time in build